needle-notion-connector/.env.example

```diff
@@ -0,0 +1,8 @@
+DATABASE_URL="postgresql://postgres:postgres@127.0.0.1:5432/needle-web-connector-dev"
+NEEDLE_APP_SECRET=your-app-secret
+NEEDLE_URL=https://needle-ai.com
+
+NOTION_OAUTH_CLIENT_ID=
+NOTION_OAUTH_CLIENT_SECRET=
+NOTION_OAUTH_URL=
+NEXT_PUBLIC_APP_HOST=https://dev-tg2ugnksuita.needle-ai.com
```

Comment:
i needed it because i was first redirecting to backend then to frontend. i think i found it easier to add it here than parsing etc. but anyway as i mentioned with the access token change below, we don't need it anymore